backend/src/main/java/site/coduo/referencelink/domain/Category.java

```diff
@@ -14,11 +14,15 @@ public class Category {
 
     public Category(final String value) {
         validate(value);
-        this.value = value;
+        this.value = value.trim();
     }
 
     private void validate(final String value) {
-        if (value.length() > CATEGORY_NAME_MAX_LENGTH) {
+        if (value == null || value.isBlank()) {
+            throw new InvalidCategoryException("카테고리 값이 비어 있습니다.");
+        }
+
+        if (value.trim().length() > CATEGORY_NAME_MAX_LENGTH) {
             throw new InvalidCategoryException("카테고리 길이는 " + CATEGORY_NAME_MAX_LENGTH + "자 이하여야 합니다.");
```

Comment:
[제안]
프로젝트 코드 전반적인 일관성을 위해서 아래와 같이 바꾸는 것이 어떨까요? 
```suggestion
        if (value.trim().length() > CATEGORY_NAME_MAX_LENGTH) {
            throw new InvalidCategoryException(String.format("카테고리 길이는 %d자 이하여야 합니다."));
```